tools/packaging/deploy/enclave-cc-deploy.sh

```diff
@@ -25,6 +25,16 @@ function print_usage() {
 	echo "Usage: $0 [install/cleanup/reset]"
 }
 
+function create_runtimeclass() {
+	echo "Creating the runtime classes"
+	kubectl apply -f runtimeclass/enclave-cc.yaml
```

Comment:
this would work:
```suggestion
	kubectl apply -f /runtimeclass/enclave-cc.yaml
```

Reply:
Changed here and also in the delete part.